js/index.js

```diff
@@ -38,30 +38,59 @@ const siteContent = {
 };
 
 // Example: Update the img src for the logo
-let logo = document.getElementById("logo-img");
+const logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"]); 
 
-let headerImg = document.getElementById("cta-img"); 
+const headerImg = document.getElementById("cta-img"); 
 headerImg.setAttribute('src', siteContent["cta"]["img-src"]); 
 
-let middleImg = document.getElementById("middle-img"); 
+const middleImg = document.getElementById("middle-img"); 
 middleImg.setAttribute('src', siteContent["main-content"]["middle-img-src"]); 
 
 // Updated header text 
-let header1 = document.querySelector("h1"); 
+const header1 = document.querySelector("h1"); 
 header1.textContent = siteContent["cta"]["h1"]; 
 
 // Created nav bar and updated nav text 
-let navigation = document.querySelectorAll("nav a"); 
+const navigation = document.querySelectorAll("nav a"); 
 for (let i = 0; i < navigation.length; i++) {
   navigation[i].textContent = siteContent["nav"]["nav-item-" + (i + 1)]; 
   navigation[i].style.color = 'green';
 }; 
 
 // Updated button text 
-let button = document.querySelector("button");
+const button = document.querySelector("button");
 button.textContent = siteContent["cta"]["button"]; 
 
+// Updated all text within main content 
+const textContent = document.querySelectorAll("p"); 
```

Comment:
It makes sense that you need the p tag. Great job working through your blockers and getting it done.